docs/specification/1.0.md

```diff
@@ -383,35 +383,64 @@ To reduce the implementation burden on EHRs that support CDS services, CDS Hooks
+Once a CDS Service provider is selected, the EHR vendor/provider negotiates the terms under which service will be provided.  This negotiation includes agreement on patient data elements that will be prefetched and provided to the CDS Service, data elements that will be made available through an access token passed by the EHR, and steps the CDS Service must take to protect patient data and access tokens.  To enable the EHR authorization server to authorize CDS Service access to FHIR resources, the CDS Service is registered as a client to the EHR authorization server.  These business agreements are documented in the service agreement.  Because the CDS Service will receive patient data and will provide clinical decision support, the Service provider also will need to sign a Business Associate Agreement (BAA) that commits it to compliance with the Privacy and Security Rules of the Health Insurance Portability and Accountability Act (HIPAA).   
+
+Every interaction between an EHR and a CDS Service is initiated by the EHR sending a service request to a CDS Service endpoint protected using the [Transport Layer Security protocol](TLS, RFC5246)[ https://tools.ietf.org/html/rfc5246].   Through the TLS protocol the identity of the CDS Service is authenticated, and an encrypted transmission channel is established between the EHR and the CDS Service. Both the Discovery endpoint and individual CDS Service endpoints are TLS secured.
 
 ### Trusting EHRs
 
-Since the CDS Service is invoked by the EHR, the CDS Service does not have the same mechanism as EHRs to establish trust of the EHR invoking it. Signed [JSON web tokens (JWT)](https://jwt.io/) are produced by the EHR and provided to the CDS Service, allowing the CDS Service to establish trust of the calling EHR.
+The service agreement negotiated between the EHR vendor/provider and the CDS Service provider will include obligations the EHR vendor/provider commits to the CDS Service provider. Some agreements may include the use of mutual TLS, in which both ends of the channel are authenticated.  
+
+However, mutual TLS is impractical for many organizations, and because the EHR initiates the TLS channel set-up, only the CDS Service endpoint will be authenticated.  To enable the CDS Service to authenticate the identity of the EHR, CDS Hooks uses digitally signed [JSON web tokens (JWT)](https://jwt.io/).  
+
+Each time an EHR transmits a request to a CDS Service, the request MUST include an `Authorization` header presenting the JWT ` as a “Bearer” token:
+```
+Authorization:  Bearer {{JWT}}
+```
+Note that this is for every single CDS Service call, whether that be a Discovery call, a single CDS Service invocation, or multiple exchanges relating to a single service.   Also note that mutual TLS MAY be used alongside JSON web tokens to establish trust of the EHR by the CDS Service.
```

Comment:
Is it really the case that Discovery calls need to be secured with a JWT?

Reply:
@brynrhodes - Yes, our intention with the JWT was that it is present in every request to the CDS Service, including Discovery calls.

Reply:
That seems like overkill to me, but okay. I am curious what the rationale is, but I don't feel strongly about this one way or the other.

Reply:
For starters, it shouldn't be any additional work for EHRs to construct the JWT. If they are already constructing it for CDS Service requests, they can do it for Discovery requests just as easily.

Second (and the primary reason), I can see some CDS Service provider not wanting any of their endpoints being called (including Discovery) unless from a trusted EHR.